cms/templates/js/timed-examination-preference-editor.underscore

```diff
@@ -52,14 +52,6 @@
                 </label>
                 <p class='field-message' id='review-rules-description'><%- gettext('Specify any additional rules or rule exceptions that the proctoring review team should enforce when reviewing the videos. For example, you could specify that calculators are allowed.') %></p>
             </li>
-            <li class="field-checkbox field-hide-after-due">
```

Comment:
@catong It appears that this documentation did not get updated for the removal of this option as a Timed Exam-specific option.

http://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/course_features/timed_exams.html